features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplay.kt

```diff
@@ -31,22 +29,10 @@ object SessionReplay {
     fun enable(
```

Comment:
side question: while we mitigated the issue for the case when SR is initialized in the `Application#onCreate` **and** on the main thread, I believe we can still have a similar issue if SR is initialialized by the customer from the worker thread / later in the app lifecycle. Is there a way to mitigate completely?

Reply:
With current solution of `LifecycleCallback`, I don't see a solution, once user misses the `onResume` event, they need to wait until the next `onResume` to make the touch recorder work properly

Reply:
If there is no solution for such case, then maybe we should highlight in the docs indeed that SR should be initialized early.

Reply:
updated